src/panel/direction/DirectionPanel.jsx

```diff
@@ -245,7 +245,7 @@ export default class DirectionPanel extends React.Component {
     const search = buildQueryString(queryObject);
     const relativeUrl = 'routes/' + search;
 
-    window.app.navigateTo(relativeUrl, {}, {
+    window.app.navigateTo(relativeUrl, window.history.state, {
```

Comment:
A similar call exists in `RouteResult`, after an alternative route has been selected
https://github.com/QwantResearch/erdapfel/blob/master/src/panel/direction/RouteResult.jsx#L55

Don't you think it should it be updated too ?

Reply:
You're right, thanks a lot.
I forgot about the other one… I hope we can remove it soon and make these imperative url changes go through the same method, but for now let's fix it also.